### ISSUE TITLE
hollywood snap not working on Solus

### ISSUE THREAD

**Issue by MariusQuabeck** [dustinkirkland/hollywood#19]
Hey Dustin,

I've seen your talk about byobu and hollywood at UbuCon Europe in Paris last week and I've decided to try the snap :)

Unfortunately, I'm getting this when I try to start it:

`marius@Lenovo-Carbon-X1-3rd ~ $ hollywood`
`/bin/bash: error while loading shared libraries: __vdso_time: invalid mode for dlopen(): Invalid argument
` 

`marius@Lenovo-Carbon-X1-3rd ~ $ snap --version`
`snap    2.27`
`snapd   2.27`
`series  16`
`solus   3`
`kernel  4.12.12-19.current`

`marius@Lenovo-Carbon-X1-3rd ~ $ lsb_release -a`
`LSB Version:	1.4`
`Distributor ID:	Solus`
`Description:	Solus`
`Release:	3`
`Codename:	shannon`

**Comment by dustinkirkland** [comment]
Hey Marius,

Hmm, I don't recognize that error...

I've copied Sergio and Alan, who might be able to help...

Sergio?



:-Dustin
@dustinkirkland

**Comment by failathon** [comment]
Suspect glibc version or conflict - would suggest trying it on another machine, VM or container to rule out any glibc issues.   

(Solus may be making slightly different changes to glibc versions/packages compared to Ubuntu/Debian/etc.)

**Comment by dustinkirkland** [comment]
Indeed, this is broken.  I'm told by @sergiusens that he might have a fix for me in snapcraft/snapd?

**Comment by tjgillies** [comment]
solus or bust

**Comment by kz6fittycent** [comment]
It's likely because the snap is "classically" confined and is reliant on the OS's available libs. As the same libs aren't available in Solus (or other distros perhaps) including said libs in the snap build and then confining it (strict) would likely fix this.

**Comment by dustinkirkland** [comment]
Closing — the snap packaging has not been actively maintained. For the best cross-platform experience, try running via Docker:

```bash
docker run -it --rm dustinkirkland/hollywood
```
